df.item-raws.xml

```diff
@@ -863,8 +873,8 @@
 
         <int32_t name="container_capacity"/>
 
-        <stl-string name="unk_v4201_1" comment='v0.42.01'/>
-        <stl-vector name="unk_v4201_2" comment='v0.42.01'/>
+        <stl-string name="description" comment='v0.42.01'/>
+        <stl-vector name="default_improvements" comment='v0.42.01'/>
```

Comment:
Do you know what these contain? I think these might end up being `vector<void*>`s.